drakrun/drakrun/main.py

```diff
@@ -32,6 +33,7 @@
 from drakrun.networking import start_tcpdump_collector, start_dnsmasq, setup_vm_network
 from drakrun.util import patch_config, get_domid_from_instance_id
 from drakrun.vmconf import generate_vm_conf
+from drakrun.parse_xl_info import get_xl_info, get_xen_commandline
```

Comment:
Not sure if it's worth creating another file. IMO those functions could be insite `util.py`